NanoProd/crab/Run3_2022_postEE/data.yaml

```diff
@@ -2,19 +2,24 @@ config:
   params:
     sampleType: data
     era: Run3_2022_postEE
-  #lumiMask: /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions18/13TeV/Legacy_2018/Cert_314472-325175_13TeV_Legacy2018_Collisions18_JSON.txt
+  #lumiMask: /eos/user/c/cmsdqm/www/CAF/certification/Collisions22/Cert_Collisions2022_355100_362760_Golden.json
 
+EGamma_Run2022E: /EGamma/Run2022E-22Sep2023-v1/MINIAOD
```

Comment:
all 2022E should be in Run3_2022/data.yaml

Reply:
Really? In the MC campaigns it's CD and EFG for pre/postEE ([PdmV link](https://twiki.cern.ch/twiki/bin/view/CMS/PdmVRun3Analysis#Monte_Carlo)). I also know that the jet veto maps are made separately for eras CD and E ([link](https://cms-talk.web.cern.ch/t/jet-veto-maps-for-run3-data/18444)).

Reply:
hm.. we need to investigate it - depending on what global tag is recommended for E. I don't think it would be prompt as for F and G. If the GT is different from wrt CD, then we need a separate file for E.

Reply:
I am ok with your proposal. Thanks for the cross-check!